stonesoup/plotter.py

```diff
@@ -62,7 +66,8 @@ def plot_tracks(self, tracks, mapping, uncertainty=False, particle=False, track_
     def plot_sensors(self, sensors, sensor_label="Sensors", **kwargs):
         raise NotImplementedError
 
-    def _conv_measurements(self, measurements, mapping, measurement_model=None):
+    def _conv_measurements(self, measurements, mapping, measurement_model=None) -> \
+            Tuple[Dict[detection.Detection, State], Dict[detection.Clutter, State]]:
```

Comment:
This should be:
```suggestion
            Tuple[Dict[detection.Detection, StateVector], Dict[detection.Clutter, StateVector]]:
```
I'll add comment about with suggestion for import and commit them just before merging.